tests/test_expand_param_fix.py

```
@@ -0,0 +1,195 @@
+    def test_bool_true_serializes_to_capital_t(self) -> None:
+        """Python bool True becomes 'True' (capital T) in query strings.
+
+        The `requests` library calls str() on parameter values, and
+        str(True) == 'True'. Zammad requires lowercase 'true'.
+        """
+        assert str(True) == "True"
+        assert str(True) != "true"
+
+    def test_string_true_stays_lowercase(self) -> None:
+        """String "true" stays as-is in query strings."""
+        assert str("true") == "true"
```

Ruff (0.15.2)

[warning] Unnecessary `str` call (rewrite as a literal)

Replace with string literal

(UP018)